tuistupdater.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "Check if Tuist is installed"
+if command -v tuist; then
+  echo "Tuist already installed."
+  tuist update
+  (cd / && tuist version)
+else
+  bash <(curl -Ls https://install.tuist.io)
+  (cd / && tuist version)
+fi
+
+# Only fail script starting from here since 
+# `tuist update` exists with an error code although
+# it actually succeeds.
+set -e
+
+echo ""
+echo "Check if local Tuist needs to be updated"
+GLOBAL_TUIST=$(cd / && tuist version)
+echo "Global tuist version: $GLOBAL_TUIST"
+
+LOCAL_TUIST=$(cat .tuist-version)
```

Comment:
Why do we store the .tuist-version and push it to the repository if we update tuist every time anyway.

Reply:
This approach is really sick imho, great job 👍

Reply:
`.tuist-version` makes sure that we are using the same version of tuist locally and on the CI. To keep up with the latest tuist version, before running the test we check for any updates locally. This way we do not have to manually check for tuist updates.

Reply:
ic, thx